servers/partner-ci/builders/common.prepare.sh

```diff
@@ -20,22 +20,25 @@ fi
 [ -z $ISO_FILE ] && (echo "ISO_FILE variable is empty"; exit 1)
```

Comment:
remove `[ -z $ISO_FILE ] && (echo "ISO_FILE variable is empty"; exit 1)`
and
change 
`if [[ ${ISO_FILE:?} == *"Mirantis"* ]] || [[ "${FUEL_RELEASE_VERSION}" == "10" ]] `